Makefile

```diff
@@ -76,10 +76,17 @@ TCL_FORMAT ?= utils//tcl-reformat.sh
 format-tcl:
 	find . -name \*.tcl $(FORMAT_EXCLUDE) -print0 | xargs -0 -P $$(nproc) -n 1 $(TCL_FORMAT)
 
-format: format-cpp format-docs format-py format-tcl
+WS_CMD = sed -i '\''s@\s\+$$@@g'\''
```

Comment:
Can you comment what these are doing in words?  I don't have an issue with using stuff like this, but I do want it to be easily understood at a glance, and for better or worse, this doesn't quality.

Reply:
How does that look to you?